tests/test_airbase.py

```diff
@@ -7,43 +7,18 @@
 
 
 @pytest.fixture
-def client(summary_response, metadata_response) -> airbase.AirbaseClient:
+def client(metadata_response) -> airbase.AirbaseClient:
     """initialied client with mocked responses"""
     return airbase.AirbaseClient()
 
 
 @pytest.fixture()
-def all_responses(
-    summary_response, csv_links_response, csv_response, metadata_response
-):
+def all_responses(csv_links_response, csv_response, metadata_response):
     """mock responses from relevant urls"""
     return
 
 
 class TestAirbaseClient:
-    def test_init_connect_false(self, summary_response):
-        client = airbase.AirbaseClient(connect=False)
-        with pytest.raises(AttributeError):
-            client.all_countries
-        with pytest.raises(AttributeError):
-            client.all_pollutants
-        with pytest.raises(AttributeError):
-            client.pollutants_per_country
-        with pytest.raises(AttributeError):
-            client.request()
-
-        client.connect()
-        assert client.all_countries is not None
-        assert client.all_pollutants is not None
-        assert client.pollutants_per_country is not None
-        assert client.request() is not None
-
-    def test_init_connect(self, summary_response):
-        client = airbase.AirbaseClient(connect=True)
-        assert client.all_countries is not None
-        assert client.all_pollutants is not None
-        assert client.pollutants_per_country is not None
```

Comment:
Let's leave this one here just to make sure that it's being initialized from the db properly

Reply:
done